src/content/blog/ga_qas.md

```diff
@@ -0,0 +1,232 @@
+
+_**Tab 1**. Detail of each operation in GA-QAS._
+
+In the GA, some hyper-parameters need to be considered and defined before you run GA-QAS. Then, you need to create an `EEnvironment` object, the important parameter is `fitness_func`, which is the function name that we declared above:
+
+```py
+from qoop.evolution.environment import EEnvironmentMetadata
+env_metadata = EEnvironmentMetadata(
+        num_qubits, # As its name
+        depth, # Ansatz depth you want
+        num_circuit, # Number of ansatz per generation
+        num_generation, # Number of generation/iteration for GA
+        prob_mutate # Mutation probability, usually as small as 0.01 (1%)
+)
+from qoop.evolution.environment import EEnvironment
+env = EEnvironment(
+     metadata = env_metadata,
+     fitness_func = fitnessW,
+)
+env.set_filename('GAQASPrepareW')
+```
+
+Default folder result will be set as `{num_qubit}+{fitness function}+{datetime}`, if you want shorter name, use `set_filename` method.
+
+# Step 3. Run GA-QAS
+
+You can call the method `evol()` to start running GA-QAS.
+
+```py
+env.evol()
+```
+
+<!-- ```py
+env.evol(
+   verbose = 1,
+   auto_save = True
+)
+``` -->
+
+<!-- There are only two parameters: the first is verbose in running process (1 means print process bar, 0 means no print anything), and the second is the saving option.
+The saving result's folder is detailed on [GA-QAS: folder result](https://github.com/vutuanhai237/qoop/wiki/GA%E2%80%90QAS:-folder-result). -->
+
+# Step 4. Plot result
+
+Currently, we support plotting fitness values against a number of generations. In the future, we will develop more presentation plots.
+
+```py
+env.plot()
+```
+
+<figure>
+    <img src='/images/ga-qas/ga-qas_example_plot_evolution.png' style="margin: auto" alt='' />
+    <figcaption>Fig 4. Fitness values versus number of generations</figcaption>
+</figure>
+
+The result is saved in a folder; the default folder name is based on the fitness function name. We care about a file named `best_circuit.qpy`, which is our final solution. Then, we can load it by $\langle qo|op\rangle$ and put it into fitness again to test:
+
+```py
+from qoop.backend import utilities
+best_circuit = utilities.load_circuit("./GAQASPrepareW/best_circuit.qpy")
+fitness_value = fitnessW(best_circuit)
+```
+
+# Conclusion
+
+In this post, we have demonstrated how to use GA-QAS. Simply define your own problem (fitness function) and the package will help you find the best ansatz automatically. For more information and documentation, explore the following links:
+
+- $\langle qo|op\rangle$: core package for GA-QAS.
+  - [Github](https://github.com/vutuanhai237/qoop)
+  - [Wiki](https://github.com/vutuanhai237/qoop/wiki)
+  - [Paper](https://doi.org/10.1016/j.softx.2024.101726)
+- GA-QAS:
+  - Paper: Coming soon.
```

Comment:
Definitely add the paper once it's out, even once this is merged :)

Reply:
I added our recent paper on arxiv!